src/applications/claims-status/components/claim-files-tab/AddFilesForm.jsx

```diff
@@ -199,6 +199,7 @@ class AddFilesForm extends React.Component {
             name="fileUpload"
             additionalErrorClass="claims-upload-input-error-message"
             aria-describedby="file-requirements"
+            uswds="false"
```

Comment:
In this version the default for `uswds` is false so we dont have to set this here.